### ISSUE TITLE
Motion Degeneracy in Self-supervised Learning of Elevation Angle Estimation for 2D Forward-Looking Sonar. (arXiv:2307.16160v1 [cs.RO])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#27070]
Motion Degeneracy in Self-supervised Learning of Elevation Angle Estimation for 2D Forward-Looking Sonar. (arXiv:2307.16160v1 [cs.RO])<br>
https://ift.tt/uAQvpdC<br>
<p>2D forward-looking sonar is a crucial sensor for underwater robotic
perception. A well-known problem in this field is estimating missing
information in the elevation direction during sonar imaging. There are demands
to estimate 3D information per image for 3D mapping and robot navigation during
fly-through missions. Recent learning-based methods have demonstrated their
strengths, but there are still drawbacks. Supervised learning methods have
achieved high-quality results but may require further efforts to acquire 3D
ground-truth labels. The existing self-supervised method requires pretraining
using synthetic images with 3D supervision. This study aims to realize stable
self-supervised learning of elevation angle estimation without pretraining
using synthetic images. Failures during self-supervised learning may be caused
by motion degeneracy problems. We first analyze the motion field of 2D
forward-looking sonar, which is related to the main supervision signal. We
utilize a modern learning framework and prove that if the training dataset is
built with effective motions, the network can be trained in a self-supervised
manner without the knowledge of synthetic data. Both simulation and real
experiments validate the proposed method.
</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
